src/django_upgrade/fixers/null_boolean_field.py

```diff
@@ -36,7 +36,7 @@ def visit_ImportFrom(
     parents: list[ast.AST],
 ) -> Iterable[tuple[Offset, TokenFunc]]:
     if (
-        not state.looks_like_migrations_file
+        state.looks_like_models_file
```

Comment:
Good work. I think I'll actually split `looks_like_models_file` and this change into a separate PR.